addons/portal/static/src/js/portal_chatter.js

```diff
@@ -83,18 +83,12 @@ var PortalChatter = publicWidget.Widget.extend({
         // set options and parameters
         this.set('message_count', this.options['message_count']);
         this.set('messages', this.preprocessMessages(this.result['messages']));
+        // bind bus event: this (portal.chatter) and 'portal.rating.composer' in portal_rating
+        // are speparate and sibling widgets, this event is to be triggered from portal.rating.composer, 
+        // hence bus event is binded to achive usage of the event in another widget.
```

Comment:
binded => bound

achive => achieve

Reply:
the string has been changed as per suggestion